opm/parser/eclipse/EclipseState/Tables/BrineDensityTable.hpp

```diff
@@ -34,6 +34,14 @@ namespace Opm {
 
         bool operator==(const BrineDensityTable& data) const;
 
+        const std::vector<double>::const_iterator begin() const {
```

Comment:
I don't think you need to return **const** `const_iterator`s.  Regular `const_iterator`s are usually sufficient.

Reply:
right.